SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,38 @@
+header:
+  schema-version: '1.0.0'
+  expiration-date: '2025-01-25T01:00:00.000Z'
+  last-updated: '2024-01-25'
+  last-reviewed: '2024-01-25'
+  project-url: https://github.com/cilium/cilium
+  license: https://github.com/cilium/cilium/blob/main/LICENSE
+project-lifecycle:
+  status: active
+  bug-fixes-only: false
+  core-maintainers:
+    - https://github.com/cilium/cilium/blob/main/MAINTAINERS.md
+  roadmap: https://docs.cilium.io/en/latest/community/roadmap
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+dependencies:
+  third-party-packages: true
+  dependencies-lists:
+    - https://github.com/cilium/cilium/blob/main/go.mod
```

Comment:
There are other third party dependencies like k8s, optional kvstore, docker, arguably LLVM and others. What's the line here and is it concerning if this is not 100% capturing the list of dependencies?

Reply:
If there is a place to find this, it worth adding it.
https://github.com/ossf/security-insights-spec/blob/v1.0.0/specification.md#dependencies

Reply:
Ah, in that case it probably just makes sense to add the sbom.

Reply:
Sbom would be great but I didn’t find the URL

Reply:
Maybe for additional context, the SBOM is attached to the images on quay.io.